js/main.js

```diff
@@ -35,3 +35,55 @@ function getRandomInclusiveDecimal(min, max, decimalPlace) {
 }
 
 getRandomInclusiveDecimal(-2.32732428348, 10.293737474, 3);
+
+
+// Генерация данных
+
+const DATABASE = {
+  author: {
+    avatar: ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10'], // строка — адрес изображения вида img/avatars/user{{xx}}.png, где {{xx}} — это число от 1 до 10. Перед однозначными числами ставится 0. Например, 01, 02...10. Адреса изображений не повторяются.
+  },
+  offer: {
+    title: ['Шарлотка', 'Леонардо', 'Монплезир', 'Биг-мак', 'Шарлотка', 'Шарлотка', 'Шарлотка', 'Шарлотка', 'Шарлотка', 'Шарлотка'],
+    address: [location.lat, location.lng],
+    price: 3, // Случайное целое положительное число
+    type: ['palace', 'flat', 'house', 'bungalow', 'hotel'],
+    rooms: 1, // Случайное целое положительное число.
+    guests: 1, // Случайное целое положительное число.
+    checkin: ['12:00', '13:00', '14:00'],
+    checkout: ['12:00', '13:00', '14:00'],
+    features: ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'], // Значения не должны повторяться.
+    description: 'Чудесное место, где можно отведать фирменную шарлотку на Парашутной',
+    photos: ['https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg','https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg', 'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg.'],
+  },
+  location: {
+    lat: getRandomIntInclusive(35.65000, 35.70000),
+    lng: getRandomIntInclusive(139.70000, 139.80000),
+  },
```

Comment:
Нам потребуется функция придумывания места. - изготовим ее
```suggestion
//создадим в спец функции.  
```

```javascript
const generateRandomLocation = (ix)=>{  
  retrun {
    lat: getRandomPositiveFloat (35.65000, 35.70000, 1),
    lng: getRandomPositiveFloat(139.70000, 139.80000, 1),
  };
}
```

обрати внимание, что `getRandomIntInclusive` не очень подходит, все-таки координаты - они не целые числа